lib/main.coffee

```diff
@@ -28,11 +33,37 @@ module.exports =
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
+        fileDir = _path2.default.dirname(filePath)
+
+        configFile = helpers.findCached fileDir, [
```

Comment:
Agh I see. 
Will sort that this morning 

> On 29 Jul 2016, at 6:20 AM, Landon Abney <notifications@github.com> wrote:
> 
> In lib/main.coffee:
> 
> > @@ -28,11 +33,37 @@ module.exports =
> >        lint: (textEditor) =>
> >          filePath = textEditor.getPath()
> >          fileText = textEditor.getText()
> > +        fileDir = _path2.default.dirname(filePath)
> > +
> > +        configFile = helpers.findCached fileDir, [
> I was going off their documentation which says the absolute path of a configuration file can (but isn't required to) be specified using htat environment variable.
> 
> Since this PR is all about adding the configuration file support, I figured now would be a good time to implement that as well.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or mute the thread.
>